src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -339,16 +339,21 @@ def _specific_row(self, slot, market):
         return []
 
 
-class SelfSufficiency:
+class KPI:
```

Comment:
How about moving this whole class to its own module? The reason I mention this is because this class is not directly relevant to the file export endpoints, but it is about calculating area statistics/kpis